### ISSUE TITLE
Interpretation of "Live Execution Time" setting

### ISSUE THREAD

**Issue by psychemedia** [deshaw/jupyterlab-execute-time#122]
From its name, I assumed the *Live Execution Time* setting could be used to control whether the start time of a cell execution could be displayed, but that appears not to be the case?

<img width="836" alt="image" src="https://github.com/deshaw/jupyterlab-execute-time/assets/82988/8e6f8c4d-5dbb-4637-90cb-01ab12ccd45d">

It would be useful to be able to suppress that display.

**Comment by krassowski** [comment]
> I assumed the _Live Execution Time_ setting could be used to control whether the start time of a cell execution could be displayed, but that appears not to be the case

No, this setting controls whether the timer showing the execution time is shown. To give a python analogy, it control whether a `timedelta` is shown, but what your arrow is pointing to is a `datetime`. When it is enabled, it will show `Execution started at {datetime}                            {timedelta}` where `timedelta` will continue updating _live_ which may be distracting for some users (hence the opt-out setting).

Maybe the description should be updated? Do you have suggestions how to make it more clear to users? Feel welcome to open a PR, the code for it is here:

https://github.com/deshaw/jupyterlab-execute-time/blob/c020582313868db92f1b0bdbae71aeb305b29fc1/schema/settings.json#L41

If your issue is then asking about a new feature to disable showing execution start time, I wonder if this should be controlled by the same setting as controls whether to show the date after the execution is complete:

![image](https://github.com/deshaw/jupyterlab-execute-time/assets/5832902/022cd608-1fa5-460f-8e43-99608159d5b8)

Disabling that setting currently makes the following difference:

| Enabled | Disabled |
|---|--|
| ![image](https://github.com/deshaw/jupyterlab-execute-time/assets/5832902/e1a76163-1139-4ea2-9d79-dd8960151efb) | ![image](https://github.com/deshaw/jupyterlab-execute-time/assets/5832902/d2d6ed01-1cce-46d6-afa5-5f27d28613a2) |

or does it make sense to make it a separate setting? We also have a failed state which shows the start datetime, see:
- ![image](https://github.com/deshaw/jupyterlab-execute-time/assets/5832902/d4303c06-b3bd-442f-aee3-6aea43daee5d)
- ![image](https://github.com/deshaw/jupyterlab-execute-time/assets/5832902/6729c20e-ace6-4891-bf63-1702cdc3ee1f)
- ![image](https://github.com/deshaw/jupyterlab-execute-time/assets/5832902/0addfe1b-9c3e-499b-b6c3-eb1732ec3e39)

**Comment by psychemedia** [comment]
ah, ok with the interpretation. I hadn't noticed the timer updating live (maybe I was only checkiung that setting on long running cells when it was off).

I'm offline and off-grid now-ish till next week. Will try to contribute back into docs when I get back.

Re: new feature - yes, essentially a combination of settings that lead to a similar UI effect (at least) of disabling the extension and not displaying anything. (If all outputs are disabled, I guess the timer doesn't actually need to start at all.)

My use case is: we are exploring a student facing JupyterLab environment where student users do not have permission to enable/disable/uninstall certain extensions. That said, if students *really* want to disable features that we have enabled by default, via a user setting, it's handy for them to be able to do so.
